concordium-consensus/testdata/contracts/v1/checkpointing.wat

```diff
@@ -112,15 +112,30 @@
     (local.set $iter (call $state_iterate_prefix (i32.const 0) (i32.const 4)))
     (call $assert_eq_64 (i64.const 0) (local.get $iter)) ;; the first iter will have id 0.
 
-    ;; we do a transfer if we were invoked with an amount of 1234.
+    ;; We carry out a transfer if this contract is invoked with an amount of '1234'.
     (if (i64.eq (i64.const 1234) (local.get $amount))
         (then (local.set $rv (call $invoke (i32.const 0) (i32.const 0) (i32.const 40))))
         (else (local.set $rv (call $invoke (i32.const 1) (i32.const 0) (call $get_parameter_size (i32.const 0))))))
-        
+
+    ;; Otherwise we do a contract invocation.
+    ;; If this contract was invoked with '10000000' then it is expected that the contract invocation is:
```

Comment:
I would probably prefer if these amounts were smaller and easier to read, e.g., `1, 2, 3, 4, ...`.

I don't see a need for so many 0's that are easy to count wrong :)